src/DeclarationProvider.ts

```diff
@@ -231,11 +263,11 @@ export class DeclarationProvider implements Disposable {
                 continue;
             }
 
-            // //VAR
-            match = /^\s*(?:m\.)([a-zA-Z_0-9]*)/i.exec(text);
+            // //FIELD
+            match = /^(?!.*\()(?: |\t)*(public|private)(?: |\t)*([a-z|\.|_]*).*((?: |\t)*=(?: |\t)*.*)*$/i.exec(text);
```

Comment:
Missed `protected` here.

Reply:
oh. I didn't realize fields could be protected. thanks

Reply:
addressed